pkg/logqlmodel/stats/context.go

```diff
@@ -168,7 +171,8 @@ func (i *Ingester) Merge(m Ingester) {
 func (r *Result) Merge(m Result) {
 	r.Querier.Merge(m.Querier)
 	r.Ingester.Merge(m.Ingester)
-	r.ComputeSummary(time.Duration(int64((r.Summary.ExecTime + m.Summary.ExecTime) * float64(time.Second))))
+	r.ComputeSummary(time.Duration(int64((r.Summary.ExecTime+m.Summary.ExecTime)*float64(time.Second))),
```

Comment:
Not required for this PR, but I wonder why we store `execTime` as seconds instead of nanoseconds 🤔

Reply:
Yes, I agree that this could also be stored as nanoseconds. This way it would also stay consistent with the `queueTime`. I can address this in a separate PR 👍

Reply:
Here it is the associated PR: https://github.com/grafana/loki/pull/5034